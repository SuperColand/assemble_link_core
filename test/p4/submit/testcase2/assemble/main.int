.data
20

.text
lui $at arr@Hi
ori $t1 $at arr@Lo
addiu $t0 $0 45
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9216
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 4198
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7795
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9484
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9650
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 4590
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6431
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 705
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8316
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 5557
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8189
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2652
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 606
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2153
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 7829
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9813
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 367
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6658
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8961
sw $t0 0 $t1
addiu $t1 $t1 4
lui $at arr@Hi
ori $a0 $at arr@Lo
addiu $a1 $0 20
jal bubble
syscall
