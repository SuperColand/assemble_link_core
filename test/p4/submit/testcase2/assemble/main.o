3c010000
34290000
2408002d
ad280000
25290004
24082400
ad280000
25290004
24081066
ad280000
25290004
24081e73
ad280000
25290004
2408250c
ad280000
25290004
240825b2
ad280000
25290004
240811ee
ad280000
25290004
2408191f
ad280000
25290004
240802c1
ad280000
25290004
2408207c
ad280000
25290004
240815b5
ad280000
25290004
24081ffd
ad280000
25290004
24080a5c
ad280000
25290004
2408025e
ad280000
25290004
24080869
ad280000
25290004
24081e95
ad280000
25290004
24082655
ad280000
25290004
2408016f
ad280000
25290004
24081a02
ad280000
25290004
24082301
ad280000
25290004
3c010000
34240000
24050014
0c000c43
0000000c
24080000
00058021
2610ffff
11100012
24090000
240a0000
02088823
1131000c
254b0004
008a6821
008b7021
8db20000
8dd30000
02726023
05810002
add20000
adb30000
25290001
254a0004
08000c4a
25080001
08000c46
03e00008
