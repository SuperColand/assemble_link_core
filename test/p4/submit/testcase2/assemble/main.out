.data
20

.text
3c010000
34290000
2408002d
ad280000
25290004
24082400
ad280000
25290004
24081066
ad280000
25290004
24081e73
ad280000
25290004
2408250c
ad280000
25290004
240825b2
ad280000
25290004
240811ee
ad280000
25290004
2408191f
ad280000
25290004
240802c1
ad280000
25290004
2408207c
ad280000
25290004
240815b5
ad280000
25290004
24081ffd
ad280000
25290004
24080a5c
ad280000
25290004
2408025e
ad280000
25290004
24080869
ad280000
25290004
24081e95
ad280000
25290004
24082655
ad280000
25290004
2408016f
ad280000
25290004
24081a02
ad280000
25290004
24082301
ad280000
25290004
3c010000
34240000
24050014
0c000000
0000000c

.symbol
0	%arr
0	main

.relocation
0	arr@Hi
4	arr@Lo
248	arr@Hi
252	arr@Lo
260	bubble
